helm/nginx-ingress-controller-app/values.yaml

```diff
@@ -529,7 +534,6 @@ controller:
 
     # -- Use an existing PSP instead of creating one
     existingPsp: ""
-    networkPolicyEnabled: true
```

Comment:
Should we mention this as a breaking change? In case a customer is using this setting it needs to be changed during the update, right?

Reply:
Exactly, gonna add a note to the changeling.